### PR TITLE
Add content for developers extending their module compatibility range to PS 1.6

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
This PR gives warnings about issues we met while making our new modules compatible with PS 1.6.

### CODE REVIEW

File: src/content/1.7/modules/core_updates/1.6.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Backward compatibility with PrestaShop 1.6
+menuTitle: Compatibility with 1.6
+---
+
+# Backward compatibility with PrestaShop 1.6
+
+If you plan to develop a module for PrestaShop 1.7 then extending the compaitibility with previous major versions of PS,
```
Comment (PierreRambaud) [inline src/content/1.7/modules/core_updates/1.6.md]:
```suggestion
If you plan to develop a module for PrestaShop 1.7 then extending the compatibility with previous major versions of PS,
```

File: src/content/1.7/modules/core_updates/1.6.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+it may require some changes before being ready for production.
+
+## Namespaces
+
+PrestaShop 1.6 does not fully support namespaces. They throw some issues when used in specific places.
+
+* In the main class of your module, the keyword `use [...];` will trigger syntax errors when PrestaShop will try to parse the file.
+* ObjectModels can't be defined in a namespace. The hooks generated while managing this entities will be considered as invalid by the `Validate` class and will trigger fatal errors.
```
Comment (NeOMakinG) [inline src/content/1.7/modules/core_updates/1.6.md]:
```suggestion
* ObjectModels can't be defined in a namespace. The hook generated while managing this entity will be considered as invalid by the `Validate` class and will trigger fatal errors.
```

Or plural it as you prefer

**Comment by matks** [comment]
Applied suggested feedbacks into new commit. Thanks @Quetzacoalt91

**Comment by Quetzacoalt91** [comment]
Oh thanks for applying the changes, I totally missed the review!

**Pull request merged**